clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -831,6 +831,40 @@ uint64_t3 reversebits(uint64_t3);
 _HLSL_BUILTIN_ALIAS(__builtin_elementwise_bitreverse)
 uint64_t4 reversebits(uint64_t4);
 
+//===----------------------------------------------------------------------===//
+// frac builtins
+//===----------------------------------------------------------------------===//
+
+/// \fn T round(T x)
+/// \brief Rounds the specified value to the nearest integer. Halfway cases are
+/// rounded to the nearest even. \a x parameter. \param x The specified input
```

Comment:
```suggestion
/// rounded to the nearest even value.
/// \param x The specified input
```

Typically `\param` command starts on a new line.

Reply:
i see clang format was doing this. I'm change this sting up a bit so that it doesn't collapse the lines.